tests/tasks/setup_mock_wifi_wpa3_owe.yml

```diff
@@ -0,0 +1,96 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: install hostapd and mac80211_hwsim kernel module in RHEL 8
+  shell: |
+    yum-config-manager --add-repo http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/BaseOS/$(uname -p)/os
+    rpm --import http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/BaseOS/$(uname -p)/os/RPM-GPG-KEY-redhat-release
+    yum-config-manager --add-repo http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/AppStream/$(uname -p)/os/
+    rpm --import http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/AppStream/$(uname -p)/os/RPM-GPG-KEY-redhat-release
+    dnf -y install java-headless
+    cd /etc/yum.repos.d/
+    rpm -i http://hdn.corp.redhat.com/rhel7-csb-stage/RPMS/noarch/redhat-internal-cert-install-0.1-23.el7.csb.noarch.rpm
+    curl -L -O https://download.devel.redhat.com/rel-eng/RCMTOOLS/rcm-tools-rhel-8-baseos.repo
+    dnf -y -y install brew-tools
+    cd /root/
+    brew download-build --arch=$(uname -p) kernel-modules-internal-$(uname -r)
+    dnf -y install kernel-modules*.rpm
+    dnf -y copr enable liangwen12year/hostapd-owe
+    dnf -y install hostapd
```

Comment:
Please try to use code in `tasks/setup_mock_wifi.yml` instead of duplicate it.

Reply:
actually, `tests/tasks/setup_mock_wifi_wpa3_owe.yml` and `tasks/setup_mock_wifi.yml` differs significantly

Reply:
I created a new file `tests/tasks/start_mock_wifi.yml` which contains the reusable code

Reply:
@cathay4t , and I also created a bugzilla(https://bugzilla.redhat.com/show_bug.cgi?id=1937537) to request WPA3 (CONFIG_OWE=y and CONFIG_SAE=y) support by default in hostapd Fedora and EPEL8 rpm.  So the last step to get this PR approved is that how can we install 'mac80211_hwsim' kernel module in RHEL 8 easily ?